changelog/_unreleased/2024-12-05-don-t-consider-user-given-limits-in-storefront.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Don't consider user given limits in storefront
+issue: NEXT-37585
+---
+# Core
+* Changed the private function `getLimit` in `Shopware\Core\Content\Product\SalesChannel\Listing\Processor\PagingListingProcessor` to ignore the limit in request.
+* Changed the private function `createReviewCriteria` in `Shopware\Core\Content\Product\SalesChannel\Review\ProductReviewLoader` to ignore the limit in request.
+___
+# Storefront
+* Changed the private function `createCriteria` in `Shopware\Storefront\Page\Account\Order\AccountOrderPageLoader` to ignore the limit in request.
+* Changed the private function `createCriteria` in `Shopware\Storefront\Page\Wishlist\WishlistPageLoader` to ignore the limit in request.
```

Comment:
IMO, we don't need a changelog when we change the private function.

We just need a changelog when that causes any breaking change, new feature or you are an external developer, If not we don't need